### ISSUE TITLE
MariaDB policy?

### ISSUE THREAD

**Issue by Bwuljqh** [zendesk/maxwell#1842]
Hello,

What is the position of Maxwell and MariaDB support? I know looking at old issues that MariaDB is not fully supported but are there any plans to do so? 

Sorry to ask that there but I did not find anywhere else to do so.
Thank you

**Comment by osheroff** [comment]
yeah, it works but it still has a few bugs and so we can't really officially bless it.  I'm always happy to accept patches for maria but I haven't seen a contributor willing to step up and make it super-production-ready. 

I've got a new job where we run mariadb and once I bring maxwell in here it'll definitely become more of a 1st class citizen.

**Comment by robertsilen** [comment]
Hello, anything new regarding MariaDB? Cheers :)

**Comment by osheroff** [comment]
yeah definitely, at lastCo we put maria + maxwell into full production usage and ironed out a lot of the bugs.  There's also been plenty of bug reports from the community, I'd say Maria is "fully supported".

Having said that, the Maria folks don't mind implementing a ton of their own SQL extensions so Maxwell will hit an edge case here or there still.

**Comment by ltning** [comment]
Has GTID support been tested with MariaDB? I'm not able to start maxwell (even with an empty maxwell database) with GTIDs being enabled; the log says

```
com.github.shyiko.mysql.binlog.network.ServerException: Could not find GTID state requested by slave in any binlog files. Probably the slave state is too old and required binlog files have been purged.
```

Reverting allows maxwell to start. This is with versions 1.42.3 and 1.43.2 (tested both).

Should I open a new issue for this, or have I misunderstood how this should work?

**Comment by osheroff** [comment]
Open a new issue, this should work so something's up with your config.
